.github/workflows/python-test.yml

```diff
@@ -34,8 +34,9 @@ jobs:
       - name: Check formatting with black
         uses: psf/black@stable
         with:
-          options: "--check"
+          options: "--check --diff"
           src: "python"
```

Comment:
Should we lint files under test/e2e as well ?

Reply:
Are we not linting them ?

Reply:
Yep, because we provided source as python folder

Reply:
I see, we should lint all our python code.

Reply:
Make sense to me. @sivanantha321 can you run `black` on the `test/e2e` folder so the code is properly formatted?

Reply:
Done